privatebin/templates/deployment.yaml

```
@@ -31,6 +31,10 @@ spec:
             - name: http
               containerPort: 8080
               protocol: TCP
+          securityContext:
+            runAsUser: 65534
+            runAsGroup: 82
+            readOnlyRootFilesystem: True
```

we should probably also explicitly add these.

```suggestion
            readOnlyRootFilesystem: true
            privileged: false
            allowPrivilegeEscalation: false
```

https://kubernetes.io/docs/concepts/policy/pod-security-policy/